lib/expand.js

```diff
@@ -1020,6 +1043,7 @@ class Expander {
       if (element) {
         const newValue = this.findMatchInDataElement(element, path[0]);
         if (newValue) {
+          if(newValue)
```

Comment:
Is this a copy/paste error or something?  We shouldn't need to check `if (newValue)` twice.